packages/terra-data-grid/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Fixed
   * Fixed the column header background color for the Orion Fusion theme.
   * Fixed pinned column divider issue during keyboard resize.
+  * Fixed left column header border for components.
```

Comment:
Did you rebase with main?  This code should already be present and not show up as a change.

Reply:
This change was not made in datagrid, so I had to fix it in this PR. We're going to be inheriting styling from table from now on so the fix was needed here.

Reply:
There is no need to update the data grid logic.  We won't be using it.